client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
No, this shouldn't be used at all.. Only the basket/min/main-body-default.html should be used now